model/src/main/kotlin/config/ProviderConfiguration.kt

```diff
@@ -23,25 +23,28 @@ import com.sksamuel.hoplite.ConfigAlias
 
 import org.ossreviewtoolkit.utils.common.Plugin
 
-data class PackageCurationProviderConfiguration(
+/**
+ * The configuration of a package curation and package configuration provider.
```

Comment:
I wonder whether we should hard-code "curation" and "configuration" here, as they class might also be reused in the context of e.g. `AdviceProviders` (also see https://github.com/oss-review-toolkit/ort/pull/6613).